### PR TITLE
Added support for Knops Mini (3x2 macropad) keyboard.

### PR DESCRIPTION
**Pull request by Pawnerd**
more info about this *fabulous* keyboard can be found at www.knops.io :).

### CODE REVIEW

File: keyboards/knops/mini/keymaps/default/config.h [hunk 1]
```diff
@@ -0,0 +1,24 @@
+/* Copyright 2017 REPLACE_WITH_YOUR_NAME
```
Comment (jackhumbert) [inline keyboards/knops/mini/keymaps/default/config.h]: Might wanna update these `REPLACE_WITH_YOUR_NAME` instances :)

File: keyboards/knops/mini/readme.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+knops keyboard firmware
```
Comment (jackhumbert) [inline keyboards/knops/mini/readme.md]: Would you mind updating this file to be consistent with [our template](https://docs.qmk.fm/documentation_templates.html#keyboard-readmemd-template)?

Comment (Pawnerd) [reply keyboards/knops/mini/readme.md]: sure! I used the one generated by the add project shell script, but will modify it.

**Comment by jackhumbert** [comment]
So with the new organisational structure, your `knops_mini.[ch]` files should actually be `mini.[ch]` (a lot is done automatically with these names) - if you'd like to use `knops_mini.[ch]`, you can use one `knops_mini` folder instead of `knops/mini` :)

**Comment by Pawnerd** [comment]
I just now saw your comment about filestructures. I will change it.

File: keyboards/knops/mini/keymaps/default/Makefile [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+MIDI_ENABLE = no            # MIDI support (+2400 to 4200, depending on config)
+AUDIO_ENABLE = no           # Audio output on port C6
+UNICODE_ENABLE = no         # Unicode
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+RGBLIGHT_ENABLE = no        # Enable WS2812 RGB underlight.  Do not enable this with audio at the same time.
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+
+ifndef QUANTUM_DIR
+	include ../../../../Makefile
+endif
```
Comment (jackhumbert) [inline keyboards/knops/mini/keymaps/default/Makefile]: All Makefiles are unneeded and ignore as well :)

**Comment by Pawnerd** [comment]
I changed the naming and ignored the makefile.

**Comment by jackhumbert** [comment]
Sorry for the run-around! Didn't realise our template was out of date - just updated it :)

If you wanna delete the `Makefile`s, and update `REPLACE_WITH_YOUR_NAME` to your name, we should be good to go!

**Comment by Pawnerd** [comment]
ah damn, I just did a PR for that template error as well :)

**Comment by Pawnerd** [comment]
It should be close to perfect now :-).

**Comment by jackhumbert** [comment]
Perfect! Thanks :)

**Comment by Pawnerd** [comment]
thank you for your time and quick responses! Took some tweaking to get everything right.

**Pull request merged**